### ISSUE TITLE
Unexpected shadow on bottom of "Try the new RH extension" section

### ISSUE THREAD

**Issue by cdrage** [containers/podman-desktop#6746]
### Bug description

![Screenshot 2024-04-10 at 10 46 42 AM](https://github.com/containers/podman-desktop/assets/6422176/0dd69a55-d791-4ccc-b48d-0ebb601162d4)

Little shadow at the bottom where it was added,

### Operating system

mac M1

### Installation Method

None

### Version

1.9.0

### Steps to reproduce

View the dashboard

### Relevant log output

```shell
N/A
```

### Additional context

N/A

**Comment by benoitf** [comment]
could you rephrase the description.

Shadow is missing, shadow is there but not expected ,etc

So other people can understand what the problem is

**Comment by axel7083** [comment]
Weirdly, the dashboard layout has a `h-fit` class compared to other pages with a `h-full`. Does not seems to have been introduced by the banner as I can replicates with previous version

**Comment by cdrage** [comment]
> could you rephrase the description.
> 
> Shadow is missing, shadow is there but not expected ,etc
> 
> So other people can understand what the problem is

Shadow is not suppose to be there and is not expected
